build.sbt

```diff
@@ -456,6 +469,12 @@ lazy val `scio-core`: Project = project
       "org.apache.beam" % "beam-runners-spark" % beamVersion % Provided exclude (
         "com.fasterxml.jackson.module", "jackson-module-scala_2.11"
       ),
+      "org.apache.beam" % "beam-runners-flink-1.9" % beamVersion % Provided excludeAll (
```

Comment:
Wait so they only publish `FlinkRunner` for 2.11?

Reply:
indeed!

Reply:
Does it mean that it will only work with scala 2.11 ?

Reply:
it works as well with 2.12 ... The job I showed you as running on a 2.12 cluster